pyswarms/single/local_best.py

```diff
@@ -251,12 +251,8 @@ def optimize(self, objective_func, iters, fast=False, **kwargs):
             )
         # Obtain the final best_cost and the final best_position
         final_best_cost = self.swarm.best_cost.copy()
-        final_best_pos = self.swarm.best_pos.copy()
-        # Write report in log and return final cost and position
-        self.rep.log(
-            "Optimization finished | best cost: {}, best pos: {}".format(
-                final_best_cost, final_best_pos
-            ),
-            lvl=20,
```

Comment:
@ljvmiranda921 which logging is right here? The one that I removed or the one that I added? It was a merge conflict and I just used the one from my commits that may be wrong :confused: